src/components/home/oldStore.js

```diff
@@ -0,0 +1,81 @@
+import { View, Text, StyleSheet, ImageBackground, Dimensions, TouchableOpacity, ScrollView, Image } from 'react-native'
+import { useTranslation } from 'react-i18next'
+
+import useGameContext from 'contexts/Game'
```

Comment:
don't know if we should keep this, do we really want custom map markers? what do you think? ping me on messenger. if not then just remove this entirely

Reply:
Nah don't think so, just removed it so it doesn't tempt you anymore 👀